### PR TITLE
Home page copy

### PR DESCRIPTION
**Pull request by AydinHassan**

### CODE REVIEW

File: _src/install.hbs [hunk 1]
```diff
@@ -100,25 +100,25 @@ description: "Installation instructions for PHPSchool"
...
-                    <li>1. Download and run the installer over at <a href="https://getcomposer.org/doc/00-intro.md#using-the-installer">https://getcomposer.org/doc/00-intro.md#using-the-installer</a></li>
-                    <li>2. Restart ConEmu and then run composer to ensure it's running.</li>
-                </ul>
+                <h3>Step 3: Check if Composer is installed</h3>
+                <ol>
+                    <li>Download and run the installer over at <a href="https://getcomposer.org/doc/00-intro.md#using-the-installer">https://getcomposer.org/doc/00-intro.md#using-the-installer</a></li>
+                    <li>Restart ConEmu and then run composer to ensure it's running.</li>
+                </ol>
 
                 <p>This site is <a href="https://github.com/php-school/php-school.github.io">open source on Github</a> & we welcome corrections/improvements.</p>
```
Comment (mikeymike) [inline _src/install.hbs]: Can we get some sort of break in here... I know it wasn't here before but it almost looks like this line is part of the instructions. 

example

Comment (mikeymike) [inline _src/install.hbs]: Maybe it should go in the footer, above the Copyright ?

**Comment by mikeymike** [comment]
@AydinHassan  @OliverChenery  Bar the one point I raised that is probably irrelevant to the PR. I'm happy with this :+1:

File: _src/install.hbs (hunk 1 above)
Comment (AydinHassan) [inline _src/install.hbs]: screen shot 2016-03-09 at 21 23 47

?

Comment (mikeymike) [inline _src/install.hbs]: yeah man looks good :+1:

**Pull request merged**